packages/contracts/contracts/PCV.sol

```diff
@@ -16,7 +16,7 @@ contract PCV is IPCV, Ownable, CheckContract, SendCollateral {
     // --- Data ---
     string constant public NAME = "PCV";
 
-    uint256 constant public BOOTSTRAP_LOAN = 10**26; // 100M thUSD
+    uint256 constant public BOOTSTRAP_LOAN = 10e26; // 100M thUSD
```

Comment:
```suggestion
    uint256 constant public BOOTSTRAP_LOAN = 10e26; // 1000M thUSD
```

If my maths are ok, 10e26 are 1000M thUSD, not 100.

Reply:
it will be 10^8 * 10^18, and 10^8 should be 100M, right?

Reply:
I mean, I think `10**26` is not the same as `10e26`.

https://docs.soliditylang.org/en/latest/types.html#rational-and-integer-literals

As far as I understand from here, `10e26` equals to `10 * 10^26`. That's `1 * 10^27` (in scientific notation). And that equals to `10^9 * 10^18`, which is `1000M`.

Does this make sense?